kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/internal/filter/NettyFilterContext.java

```diff
@@ -11,21 +11,27 @@
 import io.kroxylicious.proxy.config.PluginFactory;
 import io.kroxylicious.proxy.config.PluginFactoryRegistry;
 import io.kroxylicious.proxy.filter.FilterFactoryContext;
+import io.kroxylicious.proxy.filter.FilterThreadExecutor;
 
 import edu.umd.cs.findbugs.annotations.NonNull;
 
 public class NettyFilterContext implements FilterFactoryContext {
-    private final ScheduledExecutorService eventLoop;
+    private final FilterThreadExecutor eventLoop;
```

Comment:
As discussed elsewhere, should we change the type of the field?

I think we should change the name of the field however...